### PR TITLE
androidgcs: make work, rebrand

### PR DESCRIPTION
**Pull request by mlyle**
- remove low res resources
- detect when flight boards are plugged in, automatically get permissions to open (kinda annoying with bootloader enum but still better than without)
- scale logo artifacts

connects to #705  (just need build / packaging thought at this point, and to investigate BTLE a little).

Confirmed to work with HID to Revolution, and to OPLM.

[Review on Reviewable](https://reviewable.io/reviews/d-ronin/dronin/744)

### CODE REVIEW

**Comment by mlyle** [comment]
PFD
Launcher Prompt

File: androidgcs/res/xml/device_filter.xml [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
-	<usb-device vendor-id="8352" product-id="16732" />  
-	<usb-device vendor-id="8352" product-id="16733" />  
-	<usb-device vendor-id="8352" product-id="16734" />
+	<usb-device vendor-id="8352" product-id="16731" />  <!-- CC -->
+	<usb-device vendor-id="8352" product-id="16732" />  <!-- PIPX -->
+	<usb-device vendor-id="8352" product-id="16733" />  <!-- CC3D -->
+	<usb-device vendor-id="8352" product-id="16734" />  <!-- REVO -->
+	<usb-device vendor-id="8352" product-id="16962" />  <!-- BFPV -->
+	<usb-device vendor-id="8352" product-id="16949" />  <!-- Colibri -->
+	<usb-device vendor-id="8352" product-id="16848" />  <!-- Freedom/S1/S2 -->
```
Comment (mluessi) [inline androidgcs/res/xml/device_filter.xml:10]: this doesn't exist..

File: androidgcs/res/xml/device_filter.xml [hunk 2]
```diff
@@ -1,11 +1,18 @@
...
+
+	<usb-device vendor-id="4617" product-id="62460" />  <!-- LLUX -->
```
Comment (mluessi) [inline androidgcs/res/xml/device_filter.xml:12]: Does the first L stand for Lumenier or is this a typo?

Comment (fujin) [inline androidgcs/res/xml/device_filter.xml:12]: looks like a typo to me

Comment (fujin) [inline androidgcs/res/xml/device_filter.xml:12]: `11:31:05 <icee> heh, it was intentional`

File: androidgcs/res/xml/device_filter.xml (hunk 1 above)
Comment (mlyle) [inline androidgcs/res/xml/device_filter.xml:10]: Sparky1/Sparky2 use the ID

File: androidgcs/res/xml/device_filter.xml (hunk 2 above)
Comment (mlyle) [inline androidgcs/res/xml/device_filter.xml:12]: it stood for lumenier :P

File: androidgcs/res/xml/device_filter.xml (hunk 1 above)
Comment (mlyle) [inline androidgcs/res/xml/device_filter.xml:10]: http://www.debate.org/opinions/does-freedom-exist

Comment (mluessi) [inline androidgcs/res/xml/device_filter.xml:10]: lol

**Comment by tracernz** [comment]
Tested, works well :-D.

**Pull request merged**